symphony/lib/toolkit/class.authormanager.php

```diff
@@ -81,6 +81,43 @@ public static function delete($id)
             ->success();
     }
 
+    /**
+     * Undocumented function
```

Comment:
"Undocumented function"? Looks like an oversight to me.

Reply:
Ah, you did it later!

Reply:
;)